liberty-maven-plugin/src/main/resources/META-INF/plexus/components.xml

```diff
@@ -56,6 +56,9 @@
                             <post-integration-test>
                                 net.wasdev.wlp.maven.plugins:liberty-maven-plugin:stop-server
                             </post-integration-test>
+                            <verify>
+                               org.apache.maven.plugins:maven-failsafe-plugin:2.20:verify
```

Comment:
Also change the version of the failsafe plugin from 2.19.1 to 2.20 in line 39.  I forgot to mention that this version had changed.